### ISSUE TITLE
Link header for pod root index.html is null

### ISSUE THREAD

**Issue by jeff-zucker** [solid/node-solid-server#1387]
This script prints "null"
```javascript
const auth = require('solid-auth-cli')
const root = "https://jeffz.solid.community/index.html"

auth.login().then(
    auth.fetch( root ).then( response =>
        console.log( response.headers.get("link") )
    )
)

```

**Comment by jaxoncreed** [comment]
That's strange. It should return a header. I don't remember if headers.get cares about case sensitivity, but it does return a "Link" header rather than a "link" header.

**Comment by csarven** [comment]
As I understand it in NSS, /index.html is a resource contained in /.

/index.html is also purposed as a representation of /.

/index.html doesn't really need an LDP interaction model if that's what you're looking for, but I think it is generally expected to behave as a generic ldp:Resource here.

**Comment by jeff-zucker** [comment]
@jaxoncreed Nope, the capitalization of link/Link does not matter.  If I ask for /foo/, I get link headers either way and if I ask for /foo/index.html I get no headers either way.  
```javascript
const auth = require('solid-auth-cli')
const file = "https://jeffz.solid.community/index.html"
const path = "https://jeffz.solid.community/"

auth.login().then(
    auth.fetch( file ).then( response => {
        console.log( response.headers.get("link") )
        console.log( response.headers.get("Link") )
    }).then(
        auth.fetch( path ).then( response => {
            console.log( response.headers.get("link") )
            console.log( response.headers.get("Link") )
        })
    )
)
```
Outputs :
```
null
null
<.acl>; rel="acl", <.meta>; rel="describedBy", <http://www.w3.org/ns/ldp#Resource>; rel="type"
<.acl>; rel="acl", <.meta>; rel="describedBy", <http://www.w3.org/ns/ldp#Resource>; rel="type"
```

**Comment by jeff-zucker** [comment]
@csarven I see the theoretical benefit of treating index.html as a representation, but it's going to be quite confusing if it isn't also treated as a resource.  I can currently GET index.html and expect it to have all its headers.  I'm looking at this from the context of a client app wanting to copy all contents of a pod.  Would a client app have to query / for a variety of representations to know what to copy?

**Comment by melvincarvalho** [comment]
@jeff-zucker good catch!

I also deploy apps to index.html, and would like to be able to read and write to that resource, so I have upvoted this issue

If time allows I may be able to help

Do we have a desired behaviour?
